### PR TITLE
Participations view data dictionary.

### PR DESCRIPTION
**Pull request by ramillim**
Final content is pending the merging of the promoted participations PR.

Recommend viewing in the rich diff format https://github.com/mobilizeamerica/api/pull/26/files?short_path=48784ba#diff-48784baf3441d91b3e0967abfa3e6167

### CODE REVIEW

**Comment by ramillim** [comment]
Should we include a change log here or in the top level README where the API doc change log is?

File: sql_data_dictionary/README.md [hunk 1]
```diff
@@ -98,14 +95,58 @@ The `events` view includes events owned by the Organization and events owned by
...
+| created_date | timestamptz | Timestamp that the participation was created |
+| modified_date | timestamptz | Timestamp that the participation was last updated |
+| user_id | integer | Unique identifier of the User who signed up for this participation |
+| user__modified_date | timestamptz | Timestamp the User was last updated |
+| user__given_name | varchar(255) | User's current first name |
+| user__family_name | varchar(255) | User's current last name |
+| user__email_address | citext | User's current email address |
+| user__phone_number | varchar(15) | User's current phone number |
+| user__postal_code | varchar(10) | User's current zip code |
+| event_id | integer | Foreign Key to the related Event |
```
Comment (pchien) [inline sql_data_dictionary/README.md]: this is also null when firewalled

File: sql_data_dictionary/README.md [hunk 2]
```diff
@@ -98,14 +95,58 @@ The `events` view includes events owned by the Organization and events owned by
...
+| timeslot_id | integer | Foreign Key to the related Timeslot. This is `null` if the organization and the affiliated organization are cross firewall. |
+| override_start_date | timestamptz | Time that an event's start was overridden to. This only applies for "pick a time" virtual events that were synced from VAN. |
+| override_end_date | timestamptz | Time that an event's end was overridden to. This only applies for "pick a time" virtual events that were synced from VAN. |
+| organization_id | integer | Unique identifier of the Organization that owns the event. This may be the same organization that is querying the view or an organization that it is promoting. |
```
Comment (pchien) [inline sql_data_dictionary/README.md]: when firewalled you can only see `organization` or `affiliation` if you are that org

File: sql_data_dictionary/README.md [hunk 3]
```diff
@@ -98,14 +95,58 @@ The `events` view includes events owned by the Organization and events owned by
...
+| organization__name | varchar(100) | The public-facing name of the organization |
+| organization__slug | citext | The URL-safe string for the organization |
+| affiliation_id | integer | Unique identifier of the Organization whose feed the User signed up for the event on. This may be the same as the `organization_id` that owns the event. |
+| affiliation__name | varchar(100) | The public-facing name of the organization |
+| affiliation__slug | citext | The URL-safe string for the organization |
+| participation_status | varchar | The user's RSVP status before the event has occurred. One of: `REGISTERED`, `CANCELLED`, `CONFIRMED` |
+| attended | boolean | Whether the volunteer actually attended or not. Will be `null` if not set. |
+| experience_feedback_type | varchar | The user-reported feedback on the event. One of: `APPROVED_OF_SHIFT`, `DISAPPROVED_OF_SHIFT`, `DID_NOT_ATTEND` |
+| experience_feedback_text | text | The user-reported qualitative feedback on the event. |
```
Comment (pchien) [inline sql_data_dictionary/README.md]: also null when firewalled

File: sql_data_dictionary/README.md (hunk 1 above)
Comment (pchien) [reply sql_data_dictionary/README.md]: this is the most important one, but i think it's worth mentioning all of them though the org text is getting kinda long

Comment (ramillim) [reply sql_data_dictionary/README.md]: 👍

File: sql_data_dictionary/README.md [hunk 4]
```diff
@@ -98,14 +95,58 @@ The `events` view includes events owned by the Organization and events owned by
...
+| affiliation__name | varchar(100) | The public-facing name of the organization. This field is `null` if cross firewall unless the viewing organization is the affiliated org. |
+| affiliation__slug | citext | The URL-safe string for the organization. This field is `null` if cross firewall unless the viewing organization is the affiliated org. |
+| participation_status | varchar | The user's RSVP status before the event has occurred. One of: `REGISTERED`, `CANCELLED`, `CONFIRMED` |
+| attended | boolean | Whether the volunteer actually attended or not. Will be `null` if not set. |
+| experience_feedback_type | varchar | The user-reported feedback on the event. One of: `APPROVED_OF_SHIFT`, `DISAPPROVED_OF_SHIFT`, `DID_NOT_ATTEND` |
+| experience_feedback_text | text | The user-reported qualitative feedback on the event. This field is `null` if the viewing organization and the organization that owns the event are cross firewall. |
```
Comment (pchien) [inline sql_data_dictionary/README.md]: this is actually if the owning org and the affiliated org are cross firewall

File: sql_data_dictionary/README.md [hunk 5]
```diff
@@ -98,14 +95,58 @@ The `events` view includes events owned by the Organization and events owned by
...
+| event_id | integer | Foreign Key to the related Event. This field is `null` if the organization and the affiliated organization are cross firewall. |
+| timeslot_id | integer | Foreign Key to the related Timeslot. This field is `null` if the organization and the affiliated organization are cross firewall. |
+| override_start_date | timestamptz | Time that an event's start was overridden to. This only applies for "pick a time" virtual events that were synced from VAN. |
+| override_end_date | timestamptz | Time that an event's end was overridden to. This only applies for "pick a time" virtual events that were synced from VAN. |
+| organization_id | integer | Unique identifier of the Organization that owns the event. This may be the same organization that is querying the view or an organization that it is promoting. This field is `null` if the event owner is cross firewall. |
+| organization__name | varchar(100) | The public-facing name of the organization. This field is `null` if the event owner is cross firewall. |
+| organization__slug | citext | The URL-safe string for the organization. This field is `null` if the event owner is cross firewall. |
```
Comment (pchien) [inline sql_data_dictionary/README.md]: this is the same as affiliation, will be visible if the viewing org is the owning org (i did this just to let them know if they owned the event or drove the signup)

File: sql_data_dictionary/README.md (hunk 4 above)
Comment (ramillim) [reply sql_data_dictionary/README.md]: Ah, in my head I was thinking owning org == viewing org, but the viewing org could also be the affiliated org, so I'll change.

**Pull request merged**